global-script.lua

```diff
@@ -3982,17 +3990,24 @@ function onPlayerDisconnect(player)
 end
 
 function isPowerCard(card)
-    if card.type == "Card" and (card.hasTag("Minor") or card.hasTag("Major") or card.hasTag("Unique")) then
-        return true
-    end
-    return false
+    return card.type == "Card" and (card.hasTag("Minor") or card.hasTag("Major") or card.hasTag("Unique"))
 end
 function onObjectSpawn(obj)
     if isPowerCard(obj) then
-        applyPowerCardContextMenuItem(obj)
+        applyPowerCardContextMenuItems(obj)
     end
 end
-function applyPowerCardContextMenuItem(card)
+function applyPowerCardContextMenuItems(card)
+    card.addContextMenuItem(
```

Comment:
you should also add a hotkey for this as well

Reply:
Done.